tools/topology/topology2/platform/intel/dmic1-passthrough.conf

```diff
@@ -8,8 +8,16 @@ Object.Pipeline.host-gateway-capture [
                 Object.Widget.host-copier.1 {
                         stream_name $DMIC1_PCM_CAPS
                         pcm_id $DMIC1_PCM_ID
-                        num_input_audio_formats	2
```

Comment:
This doesn't work, need to have formats for single channels count as for dmic0. Changing to draft until fixed.

Reply:
fixed now